src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> tickets;
```

Comment:
It would be better to specify here that in your DTO's it's a list of identifiers of tickets. I mean that you have to replace tickets to ticketIds. Check common mistakes.